go.mod

```diff
@@ -31,4 +31,6 @@ require (
 	sigs.k8s.io/controller-runtime v0.8.3
 )
 
+replace github.com/codeready-toolchain/api => github.com/xcoulon/api v0.0.0-20211125125708-0f5ec2774aa7
```

Comment:
reminder

Reply:
taken care of in https://github.com/codeready-toolchain/host-operator/pull/544/commits/deae7ae645802efceabdad9ff7938e97e6a3d2d8